### PR TITLE
fix: disable prefixer, close #132

### LINKED ISSUE #132: Disable cross browser support
```css
.windi-18feqd2{
    display:-webkit-box;
    display:-ms-flexbox;
    display:-webkit-flex;
    display:flex;
}
```

to 

```css
.windi-18fe1d2 {
    display: flex;
}
```

### PR DESCRIPTION
**Pull request by hannoeru**
close #132

### CODE REVIEW

File: test/style/__snapshots__/prefixer.test.ts.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Tools / processor prefix / css / 1: |-
+closePrefixer / processor prefix / css / 1: |-
```
Comment (antfu) [inline test/style/__snapshots__/prefixer.test.ts.yml:1]: Why this entry gets changed?

Comment (hannoeru) [reply test/style/__snapshots__/prefixer.test.ts.yml:1]: Oh, because I only run that test...

Comment (hannoeru) [reply test/style/__snapshots__/prefixer.test.ts.yml]: fixed

File: test/style/prefixer.test.ts [hunk 2]
```diff
@@ -71,6 +71,7 @@ describe('closePrefixer', () => {
       prefixer: false,
     });
     expect(processor.preflight(undefined, true, true).build()).toMatchSnapshot('preflight');
-    expect(processor.interpret('bg-clip-padding flex bg-gradient-to-bl animate-ping placeholder-gray-200').styleSheet.build()).toMatchSnapshot('css');
+    // https://github.com/windicss/windicss/issues/132
```
Comment (antfu) [inline test/style/prefixer.test.ts:74]:
```suggestion
    // #132
```

**Pull request merged**